casual/casual-caller/src/main/java/se/laz/casual/connection/caller/QueueCache.java

```diff
@@ -55,6 +55,12 @@ else if (cacheMap.containsKey(queueName))
                         .filter(ConnectionFactoryEntry::isValid)
                         .collect(Collectors.toList());
 
+                if(cachedForQueue.isEmpty())
+                {
+                    LOG.info(() -> "No valid connection for queuename: " + queueName);
+                    return Optional.empty();
```

Comment:
Should there be a test to verify proper behavior when no valid entry exists for a given queue in the cache?

Reply:
Looks like there are 0 tests for QueueCache to begin with, was wondering why I had not added a test for that

Reply:
Seems it is tested via the CacheTest, I'll add a test for this as well there

Reply:
Done